src/components/mixins/DataFormat.js

```diff
@@ -0,0 +1,93 @@
+let Validator = require('validatorjs');
+
+// To include another language in the Validator with variable processmaker
+if (window.ProcessMaker && window.ProcessMaker.user && window.ProcessMaker.user.lang) {
+  Validator.useLang(window.ProcessMaker.user.lang);
+}
+
+export default {
+  props: {
+    dataFormat: {
+      type: String,
+      default() {
+        return 'string';
+      },
+    },
+  },
+  data() {
+    return {
+      validatorFormat: null,
+    };
+  },
+  watch: {
+    // Triggered whenever the v-model is updated
+    value() {
+      if (this.$parent.transientData) {
+        this.$parent.transientData[this.name] = this.formatValue(this.value);
```

Comment:
Avoid using `$parent` unless absolutely necessary; it's an escape hatch to use in rare cases, and breaks the component encapsulation that makes Vue so nice to use. If anything, `transientData` should be passed down as a prop.

Reply:
Adding **transientData** as prop